java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -36,7 +36,12 @@ abstract private class GeneratedType extends ClassOrInterface {
   }
 
   private string stubAnnotations() {
-    result = concat(stubAnnotation(this.(AnnotationType).getAnAnnotation()) + "\n")
+    result =
+      concat(Annotation an |
+        this.(AnnotationType).getAnAnnotation() = an
+      |
+        stubAnnotation(an), "\n" order by an.getType().getQualifiedName()
```

Comment:
This is actually a behavior change: Previously it would add a `\n` after each annotation. Now it only adds it between annotations but the last one does not have a trailing `\n` (see also test output changes).

When the last annotation has values (and therefore ends with `)`) that still seems to produce valid Java code, but I assume when an annotation has no values this produces malformed Java code:
```java
@MyAnnotationpublic void m() {
}
```

Reply:
Thanks @Marcono1234, good catch. I addressed this in #10662.